vita3k/modules/SceLibKernel/SceLibKernel.cpp

```diff
@@ -1016,12 +1016,38 @@ EXPORT(int, sceKernelGetLwCondInfoById) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceKernelGetLwMutexInfo) {
-    return UNIMPLEMENTED();
+EXPORT(int, sceKernelGetLwMutexInfoById, SceUID lightweight_mutex_id, Ptr<SceKernelLwMutexInfo> info) {
+    SceKernelLwMutexInfo *info_data = info.get(host.mem);
+    SceSize info_size = info_data->size;
+    if (sizeof(SceKernelLwMutexInfo) != info_size) {
+        LOG_ERROR("Unexpected SceKernelLwMutexInfo size. Expected {} Got {}", sizeof(SceKernelLwMutexInfo), info_size);
+        return SCE_KERNEL_ERROR_ILLEGAL_SIZE;
+    }
+    MutexPtr mutex = mutex_get(host.kernel, export_name, thread_id, lightweight_mutex_id, SyncWeight::Light);
+    if (mutex) {
+        info_data->uid = lightweight_mutex_id;
+        std::copy(mutex->name, mutex->name + KERNELOBJECT_MAX_NAME_LENGTH, info_data->name);
+        info_data->attr = mutex->attr;
+        info_data->pWork = mutex->workarea;
+        info_data->initCount = mutex->init_count;
+        info_data->currentCount = mutex->lock_count;
+        auto threads = host.kernel.threads;
+        for (auto it = threads.begin(); it != threads.end(); ++it) {
+            if (it->second == mutex->owner) {
+                info_data->currentOwnerId = it->first;
+                break;
+            }
+        }
+        info_data->numWaitThreads = static_cast<SceUInt32>(mutex->waiting_threads.size());
+        return SCE_KERNEL_OK;
+    } else {
+        return SCE_KERNEL_ERROR_LW_MUTEX_ERROR;
+    }
```

Comment:
same here so :) 
```suggestion
    } else
        return SCE_KERNEL_ERROR_LW_MUTEX_ERROR;
```

Reply:
We're using else { style zang.

Reply:
oh never mind github showed it in weird way.

Reply:
I would love to keep braces for all blocks even if there is only one statment inside that block. We should make this decision explicitly with `clang-format`.

Reply:
Absolutely, no matter the style, we should force it by clang-format.

Reply:
It seems that `clang-format` does not have that option. SHould we add `clang-tidy`?

- https://stackoverflow.com/q/26111162/8364403

Reply:
We can maybe talk about this in separate issue.

Reply:
More clean for me if it have only one value inside